tests/models/Attribution.test.js

```diff
@@ -121,6 +121,26 @@ const test = async app => {
     await tap.ok(response instanceof Attribution)
   })
 
+  await tap.test('Get all attributions by publicationId', async () => {
+    let attributions = await Attribution.getAttributionByPubId(publication.id)
+
+    await tap.ok(attributions)
+    await tap.ok(attributions[0] instanceof Attribution)
+    await tap.equal(attributions[0].name, 'Jane Doe')
```

Comment:
You should also check that attributions.length is correct.
You should also create multiple attributions of different types and make sure that they are all returned.